docs/src/vignettes/overview.jl

```diff
@@ -60,4 +60,9 @@ locations =
 locations[1:5]
 
 # This works because `seed` and `refine` have curried versions that can be used
-# directly in a pipeline.
\ No newline at end of file
+# directly in a pipeline.
+
+# Proposed sampling locations can then be overlayed onto the original uncertainty matrix:
+
+plt = heatmap(U; aspectratio = 1, frame = :none, c = :lapaz)
+scatter!(plt, [x[1] for x in locations], [x[2] for x in locations], ms=2.5, mc=:white, label="")
```

Comment:
I think this works:

~~~
scatter!(plt, [location.I for location in locations], ms=2.5, mc=:white, label="")
~~~